### PR TITLE
Lesson 1.8 Coding Assignment

### PR DESCRIPTION
**Pull request by azimulhuq**
Completed lesson 1.8 coding assignment.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,50 +1,134 @@
...
-          data: {
-            todoList: JSON.parse(localStorage.getItem("savedTodoList")) || [],
-          },
-        });
-      }, 2000)
-    ).then((result) => {
-      setTodoList([...result.data.todoList]);
-      setIsLoading(false);
-    });
+    fetchTodos();
```
Comment (kramduckner) [inline src/App.js:43]: if you run the app and look for loading text, you'll notice it's very brief! The way this code's built, the setIsLoading(false) will be called before the loading is finished. Async code is weird! But you'll want to setIsLoading(false) inside fetchTodos(), or await fetchTodos() and then setIsLoading(false). If you do one of those, you should see the loading... message all the way until the data from airtable gets displayed.

**Review by kramduckner — CHANGES_REQUESTED** [review]
Awesome work!!! Seriously. Just one request.

File: src/App.js (hunk 1 above)
Comment (azimulhuq) [reply src/App.js:43]: The loading was brief! Thank you. Could you look at the fetchTodos() function? I updated it as you suggested. Now it is working as expected.

**Review by kramduckner — APPROVED** [review]
Looks great! thanks for making that change.

**Pull request merged**